monocle/db/queries.py

```diff
@@ -476,11 +467,11 @@ def hot_changes(es, index, repository_fullname, params):
     _params = {
         'etype': ('Change',),
         'state': 'OPEN',
-        'repository_fullname_and_number': change_ids,
+        'change_id': change_ids,
```

Comment:
Could you change for "change_ids' as the params key ?

Reply:
done